ergo-installer.sh

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env sh
+
+# Initialize all the option variables.
+# This ensures we are not contaminated by variables from the environment.
+APP_DIR=~/Ergo
+NODE_NAME=${USER}-$(hostname)
+API_KEY=
+INSTALLATION_RECOMMENDATIONS=
+TOR=no
+CONFIG_TEMPLATE="ergo.node.mining = false\nscorex.network.nodeName = \"<NODE_NAME>\"\nscorex.restApi.apiKeyHash = \"<API_KEY_HASH>\""
+NODE_PARAMS="-Xmx$(awk '/MemTotal/ { printf "%.0f", $2*0.9 }' /proc/meminfo)K"
```

Comment:
@andyceo please add comment to explaining the formula for heap size

Reply:
Done